### PR TITLE
added hajime-scanner to net360 parser bot

### PR DESCRIPTION
**Pull request by asucrews**
Hello,

I added 5 lines to net360 parser bot to support the hajime scanner list.

Please let me know what i need to impove or change.

### CODE REVIEW

File: intelmq/bots/parsers/netlab_360/parser.py [hunk 1]
```diff
@@ -46,6 +46,11 @@ def parse_line(self, line, report):
                 event.add('destination.port', value[2].replace('dport=', ''))
                 event.add('classification.type', 'scanner')
                 event.add('classification.identifier', 'mirai', overwrite=True)
+            elif report['feed.url'] in Netlab360ParserBot.HAJIME_SCANNER_FEED:
```
Comment (wagner-certat) [inline intelmq/bots/parsers/netlab_360/parser.py]: Where is this variable defined?

Comment (asucrews) [reply intelmq/bots/parsers/netlab_360/parser.py]: Just noticed that was missing and I have updated the file.

File: intelmq/bots/BOTS [hunk 2]
```diff
@@ -347,6 +347,11 @@
             "module": "intelmq.bots.parsers.bitcash.parser",
             "parameters": {}
         },
+	"BotScout": {
```
Comment (wagner-certat) [inline intelmq/bots/BOTS]: And what's that now?

Comment (asucrews) [reply intelmq/bots/BOTS]: Lol, test for new parser i am working on....let me remove it.  sorry, not very good with git branches yet.

**Comment by wagner-certat** [comment]
> I reviewed my files and they look correct please let me know what i am missing.

Just a lack of time on my side

**Comment by wagner-certat** [comment]
Thanks, merged.

Follow-up commits: dfeffb6f9aefbf427ba1d3c0d218b2de9bb30c9f b93639517929462d44e96a9e08634b478b1492a8

**Pull request merged**